src/backend/JasmineGraphBackend.cpp

```diff
@@ -86,6 +86,7 @@ void *backendservicesesion(void *dummyPt) {
 
         } else {
             backend_logger.log("Message format not recognized", "error");
+            sleep(1);
```

Comment:
Why do we need a `sleep(1)` here?

Reply:
Will this log line be printed continuously? Shouldn't we just log the error message and exit from the loop?

Reply:
> Otherwise, it may cause an infinite loop consuming nearly 100% of a CPU thread

In the following read() it should block isn't it?

```
    while (!loop) {
        bzero(data, 301);
        read(connFd, data, 300);
```

Reply:
> In the following read() it should block isn't it?

It may not block if there is an error.

> Shouldn't we just log the error message and exit from the loop?

That will terminate the connection even on non-fatal errors, while it could have retried and continue the communication.

Reply:
Ok